src/ECER.Engines.Validation/Applications/ApplicationSubmissionValidationEngine.cs

```diff
@@ -11,8 +11,69 @@ public async Task<ValidationResults> Validate(Application application)
     await Task.CompletedTask;
 
     var validationErrors = new List<string>();
+
+    // each application should contain at least one education
+    if (!application.Transcripts.Any())
+    {
+      validationErrors.Add("the application does not have any education");
+    }
+
+    // each application should contain at least one character reference
+    if (!application.CharacterReferences.Any())
+    {
+      validationErrors.Add("the application does not have any character references");
+    }
+
+    // if the application contains the ECE Assistant certification type, the most recent education should be within 5 years
+    if (application.CertificationTypes.Any(ct => ct == CertificationType.EceAssistant))
+    {
+      var recentTranscript = application.Transcripts.FirstOrDefault();
+      if (recentTranscript != null)
+      {
+        var difference = DateTime.Today - recentTranscript.EndDate;
+        if (difference.TotalDays / 365 > 5)
```

Comment:
this will only check the first transcript, you probably need to check all of them:
`applications.Transcripts.Any(t=>t.EndDate.AddYear(5)<=DateTime.Today)`